src/big_core_input.cpp

```diff
@@ -109,29 +109,32 @@ template<typename T>
 const T* BigCoreInput::operator[](size_t index) const
 {
     // RV: tady jsem mel chybu, jako uzivatel chci ukazatel na outermost entity s indexem index (napriklad ukazatel na obrazek cislo 3), abych si z neho mohl dale brat hodnoty co potrebuju
+	// LB: tenhle return ale vrati i vsechny nasledujici obrazky
```

Comment:
A vlastne i vsechny predchozi :-) protoze od pointeru se da i odecitat. Dulezite je to, ze to rychle vrati ukazatel do pameti, kde zacina prislusny obrazek. Tady se predpoklada, ze si uzivatel zaroven vyzada rozmery a datovy typ obrazku a bude si v nem sam indexovat. Pokud si vyzada i datovy typ nasledujiciho obrazku, muze si indexovat i v nem. Cemu to vadi?